Dockerfile

```diff
@@ -1,5 +1,7 @@
 # Use OpenShift golang builder image
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.11 AS builder
+ARG BUILDER_IMAGE=${BUILDER_IMAGE:-registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.11}
+ARG TARGET_IMAGE=${TARGET_IMAGE:-registry.ci.openshift.org/ocp/4.11:base}
```

Comment:
Do we really need to duplicate the default values here ? AFAICT you'll get the default values from the Makefile.

@cpmeadors can you think of a case where this Dockerfile would be used standalone ?

Reply:
If my memory serves me right, it's required during automatic tests.

Reply:
@smuda  That rings a bell indeed. Let's add a comment in both places stating that changes should be propagated to the other file as well.

Reply:
I've added a comment about keeping them in sync.